index.js

```diff
@@ -139,8 +139,15 @@ module.exports = function(version, _options) {
             }
             name = name.replace(' (' + step.ref + ')', '');
 
-            if (name && ref && name !== ref) {
+            // In attempt to avoid using the ceremonial name of a way,
+            // check and see if the name includes strings like `highway` or `expressway`.
+            // If this is true and there is a ref, use the ref.
+            var nameIsCeremonialName = instructions[language][version].constants.ceremonialNamesToAvoid.some((stringToAvoid) => name.includes(stringToAvoid));
```

Comment:
It would be far better to look at the previous step’s `classes` for `motorway` than to rely on a word appearing in the road name. For one thing, “highway” is often used for surface streets, and even sometimes “expressway”. Moreover, in some countries, there simply aren’t good stop words that apply to only freeways.